schema.graphql

```diff
@@ -319,22 +316,6 @@ type Subgraph @entity {
   website: String!
   "Display name"
   displayName: String!
-
-  # Aggregate data, by combining all versions, and all deployments those versions point to
```

Comment:
Removed these for now, they were not in use, and a lot more thinking needs to be done if we want to include them